UltiSnips/python.snippets

```diff
@@ -709,4 +709,16 @@ snippet pmdoc "pocoo style module doc string" b
 $0
 endsnippet
 
+##################
+# Comprehensions #
+##################
+
+snippet lcp "list comprehension"
+[$1 for $2 in ${3:${VISUAL}}]$0
+endsnippet
+
+snippet dcp "dict comprehension"
+{$1: $2 for $3 in ${4:${VISUAL}}}$0
+endsnippet
```

Comment:
Remove these please, no reason to duplicate the snippets if they don't use any Ultisnips specific features :)

Reply:
Sorry, I'm not sure what you mean, what should I remove? I'm new to developing against UltiSnips, just a casual user.

Reply:
No problem! All plugins can read the snipmate format, so there's no need to duplicate snippets in both formats. The two you've added in the ultisnips format can be safely removed :)

Reply:
Thanks, mind blown 👍 

![mindblown](https://m.popkey.co/ea7a98/k8DQE_f-maxage-0_s-200x150.gif)